R/test-that.r

```diff
@@ -73,6 +82,28 @@ test_code <- function(description, code, env) {
   invisible(ok)
 }
 
+report_results <- function(results) {
```

Comment:
Maybe we should have `expectation()` (noun) to create the object, and `expect()` (verb) raise the condition?